src/rust/ide/src/view/project.rs

```diff
@@ -20,7 +19,6 @@ use nalgebra::Vector2;
 use enso_shapely::shared;
 
 
-
```

Comment:
don't remove this line

Reply:
fixed
IDE when inserting a new import consumes an empty line